lighthouse-core/audits/dobetterweb/uses-http2.js

```diff
@@ -64,10 +64,10 @@ class UsesHTTP2Audit extends Audit {
       const notH2 = /HTTP\/[01][\.\d]?/i.test(record.protocol);
       return sameHost && notH2;
     }).map(record => {
-      return Object.assign({
+      return {
         label: record.protocol,
         url: record.url // .url is a getter and not copied over for the assign.
-      }, record);
```

Comment:
You could include a few more things like contentSize, initiator, startTime... those are all safe.

up to you tho